core/Scheduler/Schedule/SpecificTime.php

```diff
@@ -30,4 +30,10 @@ public function setDay($_day)
     {
         throw new \Exception('not supported');
     }
+
+    public function setScheduledTime($scheduledTime)
+    {
+        $this->scheduledTime = $scheduledTime;
+    }
```

Comment:
We should in general try to avoid adding new methods that are "needed" in tests only. In that case you could have avoided that need by using the mock builder manually instead of `createPartialMock`.